styles/global.scss

```diff
@@ -155,7 +155,7 @@ body {
   text-overflow: ellipsis;
 }
 
-.type-table {
+.no-wrap {
```

Comment:
not needed.
there is already a global `no-wrap` class

Reply:
To make it easier for everyone to find, I have moved the `no-wrap` class from `styles/components/_top-collections.scss` to global.

Reply:
oh i see. Thanks